### ISSUE TITLE
Open in popup chat window

### ISSUE THREAD

**Issue by mahmutelmas06** [ethmtrgt/whatsend#6]
This addon opens my existing whatsapp web panel and refreshes the page if i click the button In the current situation.
This is unconveniant for most cases.

My idea is, it should open a popup chat window or a new tab if popup option is not possible. Not refrefshinc current window

**Comment by ethmtrgt** [comment]
I agree with you, this can be inconvenient. The actual reason I chose to do it this way was that a second WhatsApp Web tab opened from the same browser would override the previous one. It doesn't make much sense when the situation is like this. What do you think?
